server/routes/contacts.js

```diff
@@ -65,7 +65,7 @@ router.post('/', isAuthenticated, (req, res) => {
       twitter: body.twitter,
       instagram: body.instagram,
       github: body.github,
-      created_by: body.created_by
+      created_by: userId.created_by
```

Comment:
Is this working for you? Because you don't need the `.created_by` part. `userId`itself is just a number.